cmd/tailscaled/depaware.txt

```diff
@@ -99,7 +94,7 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
         tailscale.com/wgengine/tstun                                 from tailscale.com/wgengine
    W 💣 tailscale.com/wgengine/winnet                                from tailscale.com/wgengine/router
         golang.org/x/crypto/blake2b                                  from golang.org/x/crypto/nacl/box
-        golang.org/x/crypto/blake2s                                  from github.com/tailscale/wireguard-go/device+
+        golang.org/x/crypto/blake2s                                  from github.com/tailscale/wireguard-go/device
```

Comment:
This is also fine as-is, but FYI, if you install the latest version of depaware (as of an hour or so ago), it'll generate a smaller diff for you.

Reply:
Didn't know. Will update. Thanks!